tests/src/hipEnvVarDriver.cpp

```diff
@@ -21,6 +21,7 @@ OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWA
  * HIT_END
  */
 
+#include <io.h>
```

Comment:
This breaks CI. I am not sure why you need to include io.h in this file especially on linux.

Reply:
Hi,
Currently, hipEnvVarDriver will try to find hipEnvVar in a few places. Under directed_test first, then on the same level. If hipEnvVar doesn't exist in directed_test (but finds it in the same level), it will output error messages about a file not found, even though it's part of the searching process. I turned off stderr for this part since it is expected behavior, and only prints an error when hipEnvVar does not exist at all.

I see that io.h breaks linux, but putting guards on this include (i.e. move it to the WIN64 macro under test_common.h that detects OS) should solve this problem.
Alternatively, I can simply not turn off these error messages, but alert the user that error messages may be expected (i.e. print a message when an error may be expected so that the user is not confused)

Which approach would you say is best?